cmd/createcluster.go

```diff
@@ -105,6 +109,8 @@ func bindClusterFlags(flags *pflag.FlagSet, config *clusterConfig) {
 	flags.IntVar(&config.NumDVs, "num-validators", 1, "The number of distributed validators needed in the cluster.")
 	flags.BoolVar(&config.SplitKeys, "split-existing-keys", false, "Split an existing validator's private key into a set of distributed validator private key shares. Does not re-create deposit data for this key.")
 	flags.StringVar(&config.SplitKeysDir, "split-keys-dir", "", "Directory containing keys to split. Expects keys in keystore-*.json and passwords in keystore-*.txt. Requires --split-existing-keys.")
+	flags.StringVar(&config.LaunchpadAPIAddr, "launchpad-api-address", "https://obol-api-prod-green.gcp.obol.tech", "The launchpad api URL.")
+	flags.BoolVar(&config.Publish, "publish", false, "Publish lock file to Obol DVT launchpad.")
```

Comment:
Or, we could infer whether to publish from `config.LaunchpadAPIAddr != ""`..

Reply:
yeah indeed, not sure we should two flags, one to enable and one for the address, maybe just the address is better as v1, ie. `--publish=https://obol-api.tech`

Reply:
Discussed in discord also, Oisin keen to keep it two separate flags for now with one defaulting. Shortened to the nice URL